### DISCUSSION TITLE
Does the social play bonus work in TFT mobile?

### DISCUSSION THREAD

**Discussion by StereoSayan** [TeamFightTacticsBots/Alune#76] [Q&A] [answered]
Does the 30% increase in battle pass points apply to TFT mobile matches played?

**Comment by rexlManu** [comment, marked as answer]
yes works for tft

**Reply by StereoSayan** [reply]
Have you figured out a way to have two accounts in sync with the bot so the social play bonus applies?

**Reply by rexlManu** [reply]
i had to modify the script that it works for account that isn't the host, so i dunno
